storage/api/Storage.Samples/UploadFile.cs

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 // [START storage_upload_file]
+// [START storage_stream_file_upload]
```

Comment:
@frankyn and @shaffeeullah Same question applies here as it did for download. See https://github.com/googleapis/google-cloud-dotnet/issues/7638

(I don't know the context of the documentation in which these samples will be shown, so I cannot be sure this is correct.)

Reply:
Yep-- if this is how c# streaming works, this looks good too.